packages/models-library/src/models_library/api_schemas_dynamic_sidecar/telemetry.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from models_library.projects_nodes_io import NodeID
+from pydantic import BaseModel, ByteSize, Field
+
+
+class DiskUsage(BaseModel):
+    total: ByteSize = Field(description="total space")
+    used: ByteSize = Field(description="space used")
+    free: ByteSize = Field(description="remaining space")
+    percent: float = Field(
```

Comment:
Question: why do you need to store that value? You have everything already there no?

Reply:
It was a frontend request. Since the values are already computed and present, I'm just sending them over as is.

Reply:
TIP: since `percent == used/total` you can, depending on the user, either
a) use a property
b) add a root validator that computes `percent`

this way they **are always in sync**

Reply:
So I took the opportunity to have a better look on what psutil provides. I have refactored a a bit the model to make it more consistent with what we receive and not make it confusing to the user.